elements/openlayers-elements/test/ol-overlay.test.ts

```diff
@@ -15,7 +15,10 @@ describe('ol-overlay', () => {
       const overlay = await element.createPart()
 
       // then
-      expect(overlay.getOptions().autoPan).to.equal(true)
+      const autoPanOptions = overlay.getOptions().autoPan
+      if (typeof autoPanOptions === 'object') {
+        expect(autoPanOptions.animation?.duration).to.equal(0)
+      }
```

Comment:
You didn't like `deep.equals`? Saves you from the conditional

```suggestion
      expect(overlay.getOptions().autoPan).to.deep.equal({ animation: { duration: 0 } })
```

Reply:
Much nicer 👍 Thanks for the pointer!